llmk.lua

```diff
@@ -180,18 +192,28 @@ function M.get_status(raw)
 end
 
 -- Replace config param to filename
-function M.replace_specifiers(str, source, target)
+function M.replace_specifiers(str, source, target, output_directory)
   local tmp = '/' .. source
-  local basename = tmp:match('^.*/(.*)%..*$')
+  local basename_match = tmp:match('^.*/(.*)%..*$')
 
   str = str:gsub('%%S', source)
   str = str:gsub('%%T', target)
-
-  if basename then
-    str = str:gsub('%%B', basename)
-  else
-    str = str:gsub('%%B', source)
-  end
+  
+  local sub_basename = source
```

Comment:
`sub` being short for `substitution`, i.e. string substitution. Since the parameters were changed to include a parameter named `output_directory`, I needed to change the name of the local `output_directory` declaration to something else. I went with `sub_output_directory`, and similarly prefixed the other substitution string variables to be consistent. I am open to other suggestions for how to name these variables.